.github/scripts/build-images.sh

```diff
@@ -3,6 +3,18 @@
 # @authors: Somya Arora, Arash Pourhabibi
 # @modified: Shanqing Lin
 
+# 0. Setup QEMU and Docker buildx.
+
+# 0.1 Setup QEMU
+# reference: https://github.com/docker/setup-qemu-action/blob/master/src/main.ts
+
+docker pull 'tonistiigi/binfmt:latest'
+docker run --rm --privileged 'tonistiigi/binfmt:latest' --install arm64,riscv64
+docker run --rm --privileged 'tonistiigi/binfmt:latest'
```

Comment:
Do we need all three lines here?
I think the second line should do everything we need (including pulling the image), no?